code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -967,3 +967,74 @@
 				to_chat(M, "<span class='warning'>Your missing arm aches from wherever you left it.</span>")
 				M.emote("sigh")
 	return ..()
+
+/datum/reagent/toxin/potash
+	name = "Potash"
+	id = "potash"
+	description = "An odorless white crystal resembling and tasting of salt. Used to cause heart arrhythmias in high doses or cardiac arrest through injection. Also an effective fertiliser"
+	taste_description = "salt"
+	reagent_state = SOLID
+	color = "#FFFFFF" // rgb: 255,255,255
+	toxpwr = 0
+	metabolization_rate = 0.4
```

Comment:
You don't need to set this unless you really don't want the rate to change if someone changes the REAGENTS_METABOLISM definition (0.4, default metabolization_rate).